src/main/java/SergheiRubailo/Task9/Bank.java

```diff
@@ -0,0 +1,21 @@
+package SergheiRubailo.Task9;
```

Comment:
task 9 files should not be in this PR

Reply:
Agree, it was done by mistake. Thank you!
Unnecessary files were removed.